test/gtest/ucs/test_arbiter.cc

```diff
@@ -740,8 +750,11 @@ class test_arbiter_resched_from_dispatch : public ucs::test {
     }
 
 protected:
-    ucs_arbiter_cb_result_t dispatch(ucs_arbiter_elem_t *elem) {
-        EXPECT_EQ(&m_elem, elem);
+
+    /* the callback pushes the elem on group2 and schedules it */
+    virtual ucs_arbiter_cb_result_t dispatch(ucs_arbiter_group_t *group,
```

Comment:
why need `virtual`?

Reply:
this is actually preparation for next PR which extend the test fixture. will remove.